src/applications/debt-letters/components/DebtLetterCard.jsx

```diff
@@ -36,11 +41,9 @@ const DebtLetterCard = props => {
         <strong>Status: </strong>
         {debt.diaryCodeDescription}
       </p>
-      {debt.diaryCode && (
-        <div className="vads-u-margin-y--2 vads-u-font-size--md vads-u-font-family--sans">
-          {renderAdditionalInfo(debt.diaryCode, mostRecentHistory.date)}
-        </div>
-      )}
+      <div className="vads-u-margin-y--2 vads-u-font-size--md vads-u-font-family--sans">
+        {additionalInfo.nextStep}
```

Comment:
Could this be `undefined`?

Reply:
nah since diary codes are hardcoded there will always be a value returned and in the event the code doesn't exist to return a value the component wont be rendered anyway